### PR TITLE
Added django-rosetta, a translation package for django on Windows

### PR DESCRIPTION
**Pull request by kcemenike**
Checklist
- [✔] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [✔] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [✔] Source is from official source
- [✔] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [✔] If static libraries are linked in, the license of the static library is packaged.
- [✔] Build number is 0
- [✔] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [✔] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by kcemenike** [comment]
@conda-forge/help-python @conda-forge/staged-recipes now ready for review. 

Thanks!

File: recipes/django-rosetta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 0315a6406818c51f0f0fe04aa30328e802c8befcab1d3a34a242961284cb3387
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (scopatz) [inline recipes/django-rosetta/meta.yaml:19]: Does this need a C compiler?

File: recipes/django-rosetta/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Because it doesn't export any models, Rosetta doesn't create any tables in your project's database. 
+    Rosetta can be installed and uninstalled by simply adding and removing a single entry in your project's INSTALLED_APPS and a single line in your main urls.py file
+    Note: As of version 0.7.13 django-rosetta requires Django 1.8 or later. As of version 0.9.0, django-rosetta requires Django 1.11 or later.
+  
+  doc_url: http://django-rosetta.readthedocs.org/
+  dev_url: https://github.com/mbi/django-rosetta/
+
+extra:
+  recipe-maintainers:
+    - mbi
```
Comment (scopatz) [inline recipes/django-rosetta/meta.yaml:48]: CC @mbi - are OK with being a maintainer?

File: recipes/django-rosetta/meta.yaml (hunk 1 above)
Comment (kcemenike) [reply recipes/django-rosetta/meta.yaml:19]: I make the same mistake as in django-parler. Kindly remove please (or I will, if you need me to)

File: recipes/django-rosetta/meta.yaml (hunk 2 above)
Comment (mbi) [reply recipes/django-rosetta/meta.yaml:48]: I don't own a Windows machine, nor have ever used Conda-Forge, so I can't say I'm ready to take on this commitment  :man_shrugging:

Comment (scopatz) [reply recipes/django-rosetta/meta.yaml:48]: OK! @kcemenike - please remove @mbi from this list

File: recipes/django-rosetta/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/django-rosetta/meta.yaml:19]: Please remove it

File: recipes/django-rosetta/meta.yaml (hunk 2 above)
Comment (kcemenike) [reply recipes/django-rosetta/meta.yaml]: done, thanks

File: recipes/django-rosetta/meta.yaml (hunk 1 above)
Comment (kcemenike) [reply recipes/django-rosetta/meta.yaml]: removed

File: recipes/django-rosetta/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (scopatz) [inline recipes/django-rosetta/meta.yaml:22]: Is django not a runtime dep?

**Comment by kcemenike** [comment]
> Hi! This is the friendly automated conda-forge-linting service.
> 
> I wanted to let you know that I linted all conda-recipes in your PR (`recipes/django-rosetta`, `recipes/polib`) and found some lint.
> 
> Here's what I've got...
> 
> For **recipes/polib**:
> 
> * Feedstock with the same name exists in conda-forge.

Hi @scopatz,
I have a challenge, the required polib library is available on conda-forge, but its version for a specific OS version is less than is needed for rosetta to work.

I'm trying to add it to this PR, but I hae an error that the polib library is in the existing feedstock.

Please how can I resolve this?

**Comment by scopatz** [comment]
Please put in a PR into the polib-feedstock, updating it to the needed version.  If the maintainer of that package isn't responding then you can ping @conda-forge/core. Then you should be able to remove polib from this PR

**Comment by kcemenike** [comment]
> Please put in a PR into the polib-feedstock, updating it to the needed version. If the maintainer of that package isn't responding then you can ping @conda-forge/core. Then you should be able to remove polib from this PR

This has been done. If I don't get a response in 48h, I'll do as advised.

Thanks

**Comment by kcemenike** [comment]
@scopatz all sorted now, you may re-review.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**